lib/matplotlib/axes/_axes.py

```diff
@@ -1199,6 +1201,8 @@ def vlines(self, x, ymin, ymax, colors='k', linestyles='solid',
         ymin = self.convert_yunits(ymin)
         ymax = self.convert_yunits(ymax)
 
+        colors = rcParams['lines.color'] or 'k' if colors is None else colors
```

Comment:
I don't think `rcParams['lines.color']` can ever be something "falsy" as the validator on the rcparams ensures that it is a valid color.

Reply:
Would the better option be checking if rcParams['lines.color'] is not None?